.github/workflows/node-ci.js.yml

```diff
@@ -0,0 +1,56 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: "Run Automated Tests"
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  workflow_run:
+    workflows: ["test"]
+    types: [completed] #requested
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+    steps:
+      - name: Checkout code 🛎️
+        uses: actions/checkout@v2
+      - name: Setup Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies and run tests
+        run: |
+          npm ci
+          npm test
+          npm run coverage
+
+        # Run the coveralls action which uploads the
+        # lcov.info file
+      - name: Coveralls Parallel
+        uses: coverallsapp/github-action@master
+        with:
+          github-token: ${{ secrets.github_token }}
```

Comment:
Does this want to be `secrets.GITHUB_TOKEN` ? https://docs.github.com/en/actions/security-guides/automatic-token-authentication

Reply:
I assume yes. Not sure if the uppercase matters, but I can change that if you think so.

Reply:
> I assume yes. Not sure if the uppercase matters, but I can change that if you think so.

I think it does yes, if you could change please it that'd be great! 👍